dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoDualWriterClient.java

```diff
@@ -0,0 +1,188 @@
+package com.netflix.dyno.jedis;
+
+import com.netflix.dyno.connectionpool.ConnectionPool;
+import com.netflix.dyno.connectionpool.OperationResult;
+import com.netflix.dyno.contrib.DynoOPMonitor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import redis.clients.jedis.Jedis;
+
+import java.util.Map;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Client that provides 'dual-write' functionality. This is useful when clients wish to move from one dynomite
+ * cluster to another, for example to upgrade cluster capacity.
```

Comment:
This is a nice comment, we can reference to this line when people ask us how to scale up a cluster.